README.md

```diff
@@ -39,13 +39,21 @@ curl -L https://github.com/redhat-developer/odo/raw/master/scripts/install.sh |
 #### macOS
 
 ```sh
-sudo curl -L  "https://github.com/redhat-developer/odo/releases/download/v0.0.6/odo-darwin-amd64.gz" | gzip -d > /usr/local/bin/odo; chmod +x /usr/local/bin/odo
+# Binary installation
+sudo curl -L https://github.com/redhat-developer/odo/releases/download/v0.0.6/odo-darwin-amd64 -o /usr/local/bin/odo && chmod +x /usr/local/bin/odo
+
+# Alternative, compressed tarball installation
+sudo sh -c 'curl -L https://github.com/redhat-developer/odo/releases/download/v0.0.6/odo-darwin-amd64.gz | gzip -d > /usr/local/bin/odo; chmod +x /usr/local/bin/odo'
 ```
 
 #### Linux
 
 ```sh
-sudo curl -L  "https://github.com/redhat-developer/odo/releases/download/v0.0.6/odo-linux-amd64.gz" | gzip -d > /usr/local/bin/odo; chmod +x /usr/local/bin/odo
+# Binary installation
+sudo curl -L https://github.com/redhat-developer/odo/releases/download/v0.0.6/odo-linux-amd64 -o /usr/local/bin/odo && chmod +x /usr/local/bin/odo
```

Comment:
the binary download url says `not found`
i think we have to host the binary first before merging this

Reply:
Yup, that's why the beginning of the PR says that we must push these binaries first either this release or next release